config/routes.rb

```diff
@@ -35,13 +34,19 @@
   # Can be used by load balancers and uptime monitors to verify that the app is live.
   get "up" => "rails/health#show", as: :rails_health_check
 
-  # Defines the root path route ("/")
-  # root "posts#index"
   if Rails.env.development? || Rails.env.dev? || Rails.env.test?
     namespace :dev do
       get "/sandbox", to: "sandbox#index"
       get "/accounts", to: "accounts#index"
       post "/login", to: "accounts#login"
     end
   end
+
+  if Rails.env.test?
+    get '/', to: "dashboard#index"
```

Comment:
do we actually want this? it seems okay if we just get rid of the root path routing to the dashboard index and either remove the tests that assert it loads the dashboard or update them to assert redirected to `/dashboard`

Reply:
not a blocker, I just noticed the root path routing so it's a little funky, but I'm not sure it's better to have the test env with such a difference from other envs. I'd probably just remove any specs that test the root path, or possibly do something like mock the proxied page? 🤔

Reply:
I un-messed this